backend/tests/v5/unit/models/users.test.js

```diff
@@ -542,6 +544,7 @@ const testAddUser = () => {
 				permissions: [],
 			};
 
+			db.createUser.mockImplementationOnce(() => {});
```

Comment:
I think you can merge it with the next line? `spyOn` returns a [mock function](https://jestjs.io/docs/mock-function-api)
```suggestion
			const fn = jest.spyOn(db, 'createUser').mockImplementationOnce(() => {});
```